### DISCUSSION TITLE
New Shortcuts for Editing Math in Microsoft Word?

### DISCUSSION THREAD

**Discussion by MaryStores** [daisy/math-a11y#28] [General]
Brian and I were doing some testing when we got to wondering: What if there was a keystroke you could press that would tell you what mode you were in, such as linear or professional? The only way to know currently is to press alt, followed by j, e, c. then you determine which mode is checked and press escape several times. That's a lot of keystrokes for one sentence of information. 

Another useful keystroke might be for selecting an entire math expression.

What do you all think?

**Comment by GeorgeKerscher** [comment]
I like this! Recently we suggested to NVDA that there be single letter navigation  to math expressions and Richard suggested the = symbol. This functionality was added, but you have to assign the letter or symbol yourself. I have done this and it is terrific. I made the suggestion to our Jaws contact.

**Comment by brichwin** [comment]
To play the Word Chain memory game and include Mary's list, I think it would be helpful to have the following keyboard shortcuts:
- Say Current Equation Editor mode: (something like one of: Linear UnicodeMath, Linear LaTeX, Linear Text, Professional)
- Speak selected (to know for certain which part of an expression is selected)
- Select entire expression (to allow more confident copying of a whole expression)
- Move to the front of the expression (but still inside of the Equation Editor)
- Move to the end of the expression (but still inside the Equation Editor)
